pkg/ebpf/netlink/conntracker.go

```diff
@@ -401,5 +403,13 @@ func formatKey(c ct.Conn) (k connKey) {
 	if port, err := c.Uint16(ct.AttrOrigPortSrc); err == nil {
 		k.port = NtohsU16(port)
 	}
+	if proto, err := c.Uint8(ct.AttrOrigL4Proto); err == nil {
```

Comment:
Looks like there's a number of other options here - can we short circuit the entire entry if it's not UDP/TCP where this is called? It looks like we're storing 0 ortherwise for `k.proto`